conda/recipe/meta.yaml

```diff
@@ -60,7 +60,7 @@ requirements:
     - python
     - future >=0.17.1
     - protobuf >=3.11.2
-    - libjpeg-turbo >=1.5
+    - libjpeg-turbo >=2.0.3
```

Comment:
any reason for it to be 2.0.3 here are 2.0.4 in the Dockerfile?

Reply:
![image](https://user-images.githubusercontent.com/39967756/78971660-5f7be300-7b0c-11ea-8a2c-8ecdf465e202.png)
I can try to push conda for update upstream libjpeg-turbo.

Reply:
I have attempted to fix their build - https://github.com/conda-forge/libjpeg-turbo-feedstock/pull/16.